pulp_ansible/app/viewsets.py

```diff
@@ -401,7 +401,7 @@ def create(self, request):
 
         config, repos = self._process_config(config)
 
-        async_result = dispatch(copy_content, repos, args=[config], kwargs={})
+        async_result = dispatch(copy_content, exclusive_resources=repos, args=[config], kwargs={})
```

Comment:
since this is a content copy operation, the source repo can be among shared_resources, `_process_config` should be updated so it returns 2 lists of repos.